network/network.go

```diff
@@ -21,6 +24,8 @@ const (
 	bucketStart  = 0.001
 	bucketFactor = 2
 	numBuckets   = 5
+
+	numNeighbours = 2
```

Comment:
I thought that the lower the number here the higher is the chance that we don't ping one of the nodes. Let's say `numNeighbours = 1` and one of the nodes is down. Then its neighbour is not pinged. With `numNeighbours = 2` we are much safer in that regard. I'd put that in the comment here.

Reply:
Sure, that's a good shout.